db/migrate/20230607131645_remove_is_master_column.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveIsMasterColumn < ActiveRecord::Migration[7.0]
+  def change
+    remove_column :spree_variants, :is_master
+  end
```

Comment:
I think that this will raise lots of errors during deploy.

Reply:
Alright, I switched back to disabling the column in the class :ok_hand: